### DISCUSSION TITLE
Canny Edge Detection and Segmentation

### DISCUSSION THREAD

**Discussion by aidos2** [opengeos/leafmap#41] [Q&A] [answered]
Hello All,

Can anyone create Canny Edge Detection and Segmentation such as SNIC algorithm in GEEMAP or Leafmap?

I just converted javascript code from Google Earth Engine. But I guess in the geemap packages don't include this algorithm.
`# Perform Canny edge detection and display the result.
S2med = S2med.select(['B4', 'B3', 'B2'])
canny_may = ee.Algorithms.CannyEdgeDetector({
  'image': S2med, 'threshold': 0.1, 'sigma': 1
})
Map.addLayer(canny_may, {}, 'canny_may')`

**Comment by giswqs** [comment, marked as answer]
This question is irrelevant to leafmap. Please post GEE questions on the [geemap](https://github.com/giswqs/geemap) repo. 
Also see https://github.com/giswqs/earthengine-py-notebooks/blob/master/Image/edge_detection.ipynb
